automated_packaging/update_os_package.pl

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/perl
+use lib '/usr/local/bin';
+use common_functions;
+
+$DISTRO_VERSION = $ARGV[0];
+$PROJECT = $ARGV[1];
+$VERSION = $ARGV[2];
+
+# Name of the repo is represented differently on logs and repos
+my $github_repo_name = "citus";
+my $log_repo_name = "Citus";
+if ( $PROJECT eq "enterprise" ) {
+    $github_repo_name = "citus-enterprise";
+    $log_repo_name = "Citus Enterprise";
+}
+
+my $github_token = get_and_verify_token();
+
+# Debian branch has it's own changelog, we can get them through the CHANGELOG file of the code repo
+sub get_changelog_for_debian {
+
+    # Update project spec file
+    @changelog_file = `curl --user "$github_token:x-oauth-basic" -H "Accept: application/vnd.github.v3.raw" -X GET 'https://api.github.com/repos/citusdata/$github_repo_name/contents/CHANGELOG.md' 2> /dev/null`;
+
+    $first_version_has_seen = 0;
+    my @changelog_items;
+    foreach $line (@changelog_file) {
+        if ( $line =~ /^#/ ) {
+            if ( $first_version_has_seen == 1 ) {
+                last;
+            }
+
+            $first_version_has_seen += 1;
+        }
+        elsif ( $line =~ /^\*/ ) {
+            $line =~ tr/\`//d;
+            push( @changelog_items, '  ' . $line );
+        }
+        else {
+            push( @changelog_items, $line );
+        }
+    }
+
+    return @changelog_items;
+}
+
+# Necessary to write log date for both distros and create unique branch
+my @abbr_mon = qw(Jan Feb Mar Apr May Jun Jul Aug Sep Oct Nov Dec);
+my @abbr_day = qw(Sun Mon Tue Wed Thu Fri Sat);
+my ( $sec, $min, $hour, $mday, $mon, $year, $wday, $yday, $isdst ) =
+  localtime(time);
+$year += 1900;
+
+# Necessary to create unique branch
+$curTime = time();
+
+# Checkout the distro's branch
+`git checkout $DISTRO_VERSION-$PROJECT`;
+
+# Create a new branch based on the distro's branch
+`git checkout -b $DISTRO_VERSION-$PROJECT-push-$curTime`;
+
+# Update pkgvars
+`sed -i 's/^pkglatest.*/pkglatest=$VERSION.citus-1/g' pkgvars`;
+
+# Based on the repo, update the package related variables
+if ( $DISTRO_VERSION eq "redhat" ) {
+    `sed -i 's|^Version:.*|Version:	$VERSION.citus|g' $github_repo_name.spec`;
```

Comment:
Why did we switch from / to | as delimiter?

Reply:
To make it consistent with the following sed